.ci-operator.yaml

```diff
@@ -0,0 +1,4 @@
+build_root_image:
```

Comment:
not here! you need to add it in the https://github.com/openshift/ibm-powervs-block-csi-driver

Reply:
got it.